apps/docsite/src/scripts/splitChangelog.ts

```diff
@@ -0,0 +1,70 @@
+const fs = require("fs");
+
+const getVersionSplit = (line) => {
+  const versionString = line.substring(4, line.lastIndexOf("]"));
+  return versionString.split(".");
+};
+
+const createFreshDirectory = (directoryPath) => {
+  try {
+    if (fs.existsSync(directoryPath)) {
+      fs.rmSync(directoryPath+"/*",{ force:true })
+    }
+    fs.mkdirSync(directoryPath, { recursive: true });
+  } catch (err) {
+    console.error("Error :", err.message);
+  }
+};
+
+const splitLog = async() => {
+  const data = await fetch("https://raw.githubusercontent.com/microsoft/fluentui/master/packages/react-charting/CHANGELOG.md").then(res=>res.text()).catch((err)=>{
+    console.error("Fetch error:",err.message)
+    throw new Error(err.message)
+  });
+  const lines = data.split("\n");
+  var currentVersion = "";
+  var startChop = false;
+  var versionToLineMapping = [];
+  lines.every((line, index) => {
+    if (line.startsWith("<!-- Start content -->")) {
+      startChop = true;
+    }
+    if (startChop) {
+      if (line.startsWith("## [")) {
+        const versionSplit = getVersionSplit(line);
+        if (versionSplit[0] === "5") {
+          const minorVersion = versionSplit[0] + "." + versionSplit[1];
+          if (currentVersion != minorVersion) {
+            currentVersion = minorVersion;
+            versionToLineMapping.push({
+              version: currentVersion,
+              startLine: index,
+            });
+          }
+        } else if (versionSplit[0] === "4") {
```

Comment:
why is the else if statement needed

Reply:
That's for storing the line after which  version 5.0.0 ends